spaceship/lib/spaceship/connect_api/models/bundle_id_capability.rb

```diff
@@ -59,14 +74,26 @@ def is_type?(type)
       # API
       #
 
-      def self.all(client: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+      def self.all(client: nil, bundle_id_id: nil, filter: {}, includes: nil, limit: nil, sort: nil)
+        raise "bundle_id_id is required " if bundle_id_id.nil?
```

Comment:
I understood the previous implementation is like a copy-and-paste from `user.rb`😅  

https://developer.apple.com/documentation/appstoreconnectapi/list_all_capabilities_for_a_bundle_id
Looking at the document page, there are only a few options supported. So you can shorten the method signature like this?

```suggestion
      def self.all(client: nil, bundle_id_id:, limit: nil)
```